README.md

```diff
@@ -125,7 +125,7 @@ k[version].[type].[data]
 #### Encoding a Key
 
 ```csharp
-var paserk = Paserk.Encode(pasetoKey, type);
+var paserk = Paserk.Encode(pasetoKey, purpose, type);

```

Comment:
Seems that we might need to update this documentation to showcase the password wrap usage.